about.html

```diff
@@ -0,0 +1,38 @@
+<html>
+  <head>
+    <script src="jquery-2.1.4.min.js"></script>
+    <link rel="stylesheet" type="text/CSS" href="style.css">
+    <title>Rick's Blog</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="title">Rick Tesmond's Blog</h1>
+      <button class="hamburger"><span class="icon-menu"><img src="images/Menu.svg" /></span></button>
+      <button class="cross"><span class="icon-menu"><img src="images/Cross.svg" /></span></button>
+      <nav class="main-nav">
+        <div class="menu">
+          <ul>
+            <li><a href="index.html"><button><img class="tab" src="images/Home.png">Home</button></a></li>
+            <li><a href="about.html"><button><img class="tab" src="images/About.png">About</button></a></li>
+            <li><a href="https://www.linkedin.com/in/richardtesmond"><button><img class="tab" src="images/Linkedin.png">Linkedin</button></a></li>
+            <li><a href="https://github.com/tesmonrd"><button><img class="tab" src="images/Github.png">Github</button></a></li>
+          </ul>
+      </nav>
+    </header>
+    <main>
+      <h2>Get to Know Rick</h2>
+      <ul>
+        <li>I'm a sports addict, both watching and playing. I swim, run, play football on the weekends, and am training for a marathon. Although it is difficult, I remain an avid Bengals and Reds fan.</li>
+        <li>Live a block off of Green Lake.</li>
+        <li>Works part time for Luxe Valet (see me if you want a day of free on-demand valet parking downtown!).</li>
+      </ul>
+    </main>
+    <footer>
+
+    </footer>
+  </body>
+  <script src="article.js"></script>
+  <script src="blogArticles.js"></script>
+  <script src="events.js"></script>
+  <script src="index.js"></script>
+</html>
```

Comment:
We'd like this about section to be part of single page application rather than a separate page.